{"uuid":"14cf986e-45ed-4744-8ce8-c7155bf3b7e4","historyId":"e4b9dc95e43582900b093a57218d512","fullName":"testNG.Retry_Runner.exception","labels":[{"name":"package","value":"testNG.Retry_Runner"},{"name":"testClass","value":"testNG.Retry_Runner"},{"name":"testMethod","value":"exception"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"testNG.Retry_Runner"},{"name":"host","value":"DESKTOP-Q90AFIM"},{"name":"thread","value":"6180@DESKTOP-Q90AFIM.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"exception","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\nMethod Retry_Runner.exception()[pri:0, instance:testNG.Retry_Runner@6676f6a0] should have thrown an exception of type class java.lang.IndexOutOfBoundsException","trace":"org.testng.TestException: \nMethod Retry_Runner.exception()[pri:0, instance:testNG.Retry_Runner@6676f6a0] should have thrown an exception of type class java.lang.IndexOutOfBoundsException\r\n\tat org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:82)\r\n\tat org.testng.internal.TestInvoker.considerExceptions(TestInvoker.java:741)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:629)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\r\n\tat org.testng.TestRunner.run(TestRunner.java:587)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1109)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1039)\r\n\tat org.testng.TestNG.run(TestNG.java:1007)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1668954908370,"stop":1668954908388}